docs/getting-started-on-the-raspberry-pi.md

```diff
@@ -0,0 +1,104 @@
+# Getting Started on the Raspberry Pi 3
+
+This guide assumes you are using the [Raspbian image]. It should work on other
+distributions as well, but some details may be different.
+
+We will compile the driver on the Raspberry Pi itself, of course you can also
+cross compile from your system. 
+
+## Download Driver
+
+Get the latest release from
+https://github.com/Sensirion/embedded-uart-sps/releases (sps30-uart-x.y.z.zip)
```

Comment:
I used 1.0.0 for the rest of the guide. I'm not sure if we should use real version numbers or just `x.y.z` everywhere.

Reply:
maybe link to https://github.com/Sensirion/embedded-uart-sps/releases/latest instead?

I think using either the release number or x.y.z is fine

Reply:
Fixed.